skbio/tree/_tree.py

```diff
@@ -2705,6 +2705,47 @@ def compare_tip_distances(self, other, sample=None, dist_f=distance_from_r,
         return dist_f(self_matrix, other_matrix)
 
     @experimental(as_of="0.4.0")
+    def bifurcate(self, insert_length=None, remove_singles=True):
+        r""" Restructures tree into a bifurcating tree
+
+        All nodes that have more than 2 children will
+        have additional intermediate nodes injected to ensure that
+        every node has only 2 children
```

Comment:
Alright, so remove the `remove_singles` parameter and make it strictly bifurcating?

I wasn't sure about the precise definition.  The application that I'm developing this for requires strictly bifurcating trees.  I'm honestly not sure how useful having a bifurcating tree (i.e. every intermediate node has at least two nodes) would be.  This was the original intent on including the `remove_singles` parameter